README.md

```diff
@@ -1,80 +1,104 @@
 
+## Article IV
 
-# Article IV
-## Membership Requirements and Limitations
+### Membership Requirements and Limitations
 
 Membership to this organization requires that the student be enrolled at Clemson University full time or part time. Both undergraduate and graduate level students will be allowed to join. Members will be required to attend a minimum of 2 meetings per semester. No requirements will be established regarding major or qualifications. Membership is open to all students of Clemson University who meet the requirements listed previously.
 
+## Article V
 
-# Article V
-## Officers (Titles, Term of Office, How and When Elected)
+### Officers (Titles, Term of Office, How and When Elected)
 
-Each officer position with the exception of System Architect serves until the Spring semester election following their election. Nominations for each generally elected position, that is all positions other than System Architect, can be submitted by any member beginning one month before an election and ending at the election of the particular officer. At the last meeting of the Spring semester or the last meeting of the Fall semester, the generally elected positions that are open, that is all positions other than System Architect that are open, will be filled by a majority vote of the nominees by active members. The position of System Architect will be filled by a unanimous vote of the current officers whenever the position is open.
+Each officer position serves until the Spring semester election following their election. Nominations for each generally elected position can be submitted by any member beginning one month before an election and ending at the election of the particular officer. At the last meeting of the Spring semester or the last meeting of the Fall semester, the generally elected positions that are open will be filled by a majority vote of the nominees by active members.
 
 Officer positions are as follows:
+
 * President
 * Vice President
-* System Architect
 * Treasurer
 * Secretary
-* Webmaster
+* Competition and Events Manager
+* Training Manager
+
+Should an officer be unable to fulfill their duties due to health issues, they are relieved of their position, and their position is to be filled at the next meeting by a snap election.
+
+## Article VI
+
+### Appointed Positions (Titles, Term of Office, How Selected)
+
+Each appointed position, with the exception of Lab Administrator, serves at-will of the board of Officers, is appointed by 2/3 majority of Officers' approval, and is relieved via graduation or 2/3 majority of sitting Officers' approval. The position of Lab Administrator will be filled by a unanimous vote of the current officers and advisors, relieved only via graduation or unanimous vote of current officers and advisors.
```

Comment:
There should also be a way for an appointed officer to resign, and an appointed officer should give assent to their appointment.

Reply:
I think it's important to emphasize that these positions are _leadership_, and do not hold officer-level positions. They do not sit on the board, and are appointed

Reply:
Perhaps something like this:


>Appointed positions may be filled by any active member who is not an officer, and a member may hold more than one appointed position.
>
>Appointed positions do not have the rights or privileges as officers, and are purely functionary in nature.
>
>Each appointed position, with the exception of Lab Administrator, serves at-will of the board of Officers, is appointed by 2/3 majority of Officers' approval, and is relieved via graduation or 2/3 majority of sitting Officers' approval. The position of Lab Administrator will be filled by a unanimous vote of the current officers and advisors, relieved only via graduation or unanimous vote of current officers and advisors.

Reply:
Robert, it does not look like your version addresses the issues you outlined here:

> There should also be a way for an appointed officer to resign, and an appointed officer should give assent to their appointment.

I agree with this and think that it should be incorporated

Reply:
@AdamVanScyoc 

I was focusing on the part that @houstdav000 was focusing on.  A more complete version might be:

>Appointed positions may be filled by any active member who is not an officer, and a member may hold more than one appointed position.
>
>Appointed positions do not have the rights or privileges as officers, and are purely functionary in nature.
>
> Appointed positions may resign their position at any time by providing written notice to the Officers.
>
>Each appointed position, with the exception of Lab Administrator, serves at-will of the board of Officers, is appointed by 2/3 majority of Officers' approval, and is relieved via graduation or 2/3 majority of sitting Officers' approval. The position of Lab Administrator will be filled by a unanimous vote of the current officers and advisors, relieved only via resignation, graduation, or unanimous vote of current officers and advisors.  Once appointed, the appointed person must assent to their appointment prior to taking office.
>

Reply:
I think an adjustment like the one I made for the appointed position, covering if they cede for any reason, quitting or health. I would still include the resignation part though. 

One thing I'm not sure of, is whether an officer can or cannot hold an appointed position, and whether or not a member can hold multiple appointed positions.

A similar thing we need to add for officers is to cover that An individual can only hold one officer position.

Reply:
Officer assignment issue in #19

Reply:
Addressed resignation, and multiple assignment in most recent adjustment